src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -166,13 +166,15 @@ const ViewPopup = (() => {
             current_spot_time = user_sold ? '' : contract.exit_tick_time;
         }
 
-        if (current_spot) {
+        const is_touchnotouch_tick_contract = /touch/i.test(contract.contract_type) && contract.tick_count;
+
+        if (current_spot && !is_touchnotouch_tick_contract) {
```

Comment:
we don't want to hide it for all, only touch/no touch contracts that were sold before expiry